src/v/cluster/types.h

```diff
@@ -767,6 +767,17 @@ struct config_status_reply {
     errc error;
 };
 
+struct create_non_replicable_topics_request {
+    static constexpr int8_t current_version = 1;
+    std::vector<non_replicable_topic> topics;
+    model::timeout_clock::time_point timeout;
```

Comment:
I think this should be an `ss::lowres_system_clock::time_point`

Reply:
Chatted w/ Ben offline about this. Discussed how instead of converting between types of time units, we could instead just use a duration